src/components/UserPortal/CreateDirectChat/CreateDirectChat.tsx

```
@@ -134,9 +134,16 @@ export const handleCreateDirectChat = async (
       await createChat({
-        variables: { organizationId, userIds: [userId, id], isGroup: false },
+        variables: {
+          organizationId,
+          userIds: [userId, id],
+          isGroup: false,
+        },
       });
       await chatsListRefetch();
+      {
+        /*To be fixed*/
+      }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)